### ISSUE TITLE
Accessibility - Goods List Screen - Missing Headings

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#39]
The screen is missing headings that make navigating screen difficult for those using TalkBack (screen reader).

The problem is for all the different tabs on this screen (A –Z and By Sector).

**Comment by aquilent-dgemoets** [comment]
See discussion for #20.

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings. 

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them. 

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by aquilent-dgemoets** [comment]
I checked the repository for the (unpublished) Android version of the OSHA 'Heat Index' application and didn't see any added accessibility code/configuration.  Could you confirm that the behavior you are remembering was for the Android version of the app?

**Comment by jfc3-dol** [comment]
By removing the letters that each country start with on the "A - Z" tab removes the issue of reading order and having headings on this page.

By adding a drop down list to let people chose what region makes it work better when TalkBack reads the information on the page. It's still would be better to have headings.

**Comment by jfc3-dol** [comment]
Having headings on this page for the different sections or regions of the world would be better to help screen reader users find their way around the screen.

**Comment by carolynhuang** [comment]
This appears to be a usability feature request.  Please clarify how we can find an acceptable accessibility solution.

**Comment by jfc3-dol** [comment]
There should be headings so TalkBack users can use them to navigate the screen and find content easier.

**Comment by jfc3-dol** [comment]
Closing issue since it looks like all the headings for the sections have been removed and the application is using the drop-down list to divide the content into their proper groups.
